stdlib/ballerina-builtin/src/main/java/org/ballerinalang/nativeimpl/io/OpenFile.java

```diff
@@ -110,7 +110,8 @@ public AbstractChannel inFlow(Context context) throws BallerinaException {
                 }
             }
             FileChannel byteChannel = FileChannel.open(path, opts);
-            channel = new FileIOChannel(byteChannel, IOConstants.CHANNEL_BUFFER_SIZE);
+            // channel = new FileIOChannel(byteChannel, IOConstants.CHANNEL_BUFFER_SIZE);
```

Comment:
Shall we remove commented line

Reply:
This needs to be uncommented upon the availability of the async functions